src/store/actions.js

```diff
@@ -0,0 +1,15 @@
+import { auth } from "../firebase";
+
+const actions = {
+  AUTH_CHECK({ commit }) {
+    auth.onAuthStateChanged((user) => {
+      commit("SET_USER", user);
+    });
+  },
+
+  LOGOUT() {
+    auth.signOut();
+  },
+};
+
+export default actions;
```

Comment:
This makes the actions available to the app

Reply:
I'd say it more like: "This makes available to be `import`ed by other code"